pkg/settings/setting.go

```diff
@@ -119,6 +119,7 @@ var (
 	HideLocalCluster                    = NewSetting("hide-local-cluster", "false")
 	MachineProvisionImage               = NewSetting("machine-provision-image", "rancher/machine:v0.15.0-rancher73")
 	SystemFeatureChartRefreshSeconds    = NewSetting("system-feature-chart-refresh-seconds", "900")
+	PrometheusMetrics                   = NewSetting("prometheus-metrics", "false")
```

Comment:
using env var also makes it so the deployment needs to restart which makes it more effective, otherwise since you only gate the register, changing the setting value would have no effect anyway. It's also worth noting that once the setting val is set with an env var the value cannot be over written anyway.

Reply:
Ahh ok, good points, I'll change remove the setting and switch back to the `CATTLE_PROMETHEUS_METRICS` env var